### PR TITLE
Update font size for compact tables & Update code block theming

### PR DESCRIPTION
**Pull request by shanecmiller23**

### CODE REVIEW

**Review by jeanduplessis — APPROVED** [review]

**Comment by shanecmiller23** [comment]
@jeanduplessis apologies accidentally put the code block stuff here too. Should get another review before merge

**Comment by shanecmiller23** [comment]
Apologies, accidentally put the code block color changes too in this PR

**Review by jeanduplessis — APPROVED** [review]
LGTM 🚢

**Pull request merged**
